src/test/pages/LecturePage.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { mount, shallow } from 'enzyme';
+import './loadSubject.mock';
+import { LecturePage } from '../../main/pages/LecturePage';
+
+describe('LecturePage', () => {
+    let renderedComponent;
+
+    const props = {
+        loadSubject,
+        match: { path: '/courses/:subject_id/:subject', url: '/courses/2D0MoB57yByiAQhLSGnK/MQMO', isExact: true, params: { subject_id: '2D0MoB57yByiAQhLSGnK', subject: 'MQMO' } },
```

Comment:
also for this ```match``` property, because ```LecturePage``` component just use ```match.params```, you can provide just that ```params``` attribute for this test, the rest does not matter/no need to provide too much context for this test :)